### PR TITLE
rolling_update: fix dest path for mgr keys fetching

### PR DESCRIPTION
**Pull request by guits**
the role `ceph-mgr` that is played later in the playbook fails because
the destination path for the fetched keys is wrong.
This patch fix the destination path used in the task `fetch ceph mgr
key(s)` so there is no mismatch.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1574995

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: infrastructure-playbooks/rolling_update.yml [hunk 1]
```diff
@@ -224,7 +224,7 @@
     - name: fetch ceph mgr key(s)
       fetch:
         src: "{{ ceph_conf_key_directory }}/{{ cluster }}.mgr.{{ hostvars[item]['ansible_hostname'] }}.keyring"
-        dest: "{{ fetch_directory }}/{{ fsid }}/"
+        dest: "{{ fetch_directory }}/{{ fsid }}/{{ ceph_conf_key_directory }}"
```
Comment (yprokule) [inline infrastructure-playbooks/rolling_update.yml]: "msg": "dest is an existing directory, use a trailing slash if you want to fetch src into that directory"

Comment (guits) [reply infrastructure-playbooks/rolling_update.yml]: true. good catch, thanks @yprokule

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-update_cluster

**Comment by leseb** [comment]
jenkins test luminous-ansible2.4-update_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-centos7_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster_collocation

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-ooo_collocation

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-purge_bluestore_osds_non_container

**Pull request merged**